source/Octopus.Tentacle/Scripts/ScriptState.cs

```diff
@@ -16,21 +16,18 @@ public class ScriptState
             bool? ranToCompletion)
         {
             Created = created;
-            ScriptTicket = scriptTicket;
             Started = started;
             Completed = completed;
             State = state;
             ExitCode = exitCode;
             RanToCompletion = ranToCompletion;
         }
 
-        public ScriptState(ScriptTicket scriptTicket, DateTimeOffset created)
+        public ScriptState(DateTimeOffset created)
         {
-            ScriptTicket = scriptTicket;
             Created = created;
         }
-
-        public ScriptTicket ScriptTicket { get; }
```

Comment:
Although this was from a direction that was backed away from, `ScriptTicket` was never used (not even in Octopus Server).

So I kept this, as it wasn't wasted work.

Reply:
Just checking that removing this won't cause any issues if Tentacle loads up the ScriptState that already has the ScriptTicket property.

Reply:
It's just JSON deserialization, so any extra properties in the JSON should be ignored.

But the proof is in the pudding 😁 

So this JSON (which includes the script ticket) loaded successfully into the new object (which does not contain script ticket field anymore)
```
{"ScriptTicket":{"TaskId":"ServerTasks-12881-5e494d62-83fa-4229-9292-05f0fcfb91f5"},"Created":"2023-09-14T03:49:05.5241924+00:00","Started":"2023-09-14T03:49:05.5752389+00:00","Completed":"2023-09-14T03:49:07.8029394+00:00","State":2,"ExitCode":0,"RanToCompletion":true}
```
![image](https://github.com/OctopusDeploy/OctopusTentacle/assets/91929764/dee2f67c-144b-47d8-bbff-8bf4003826aa)